CppProperties.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Looks like these two files are not related to the PR.

Reply:
Thank you, fixed.  FYI, the changes to the fmod include in two files were needed to build and test on my machine, at least for now.  I'm not sure they're required for someone more experienced.